runtime/j9vm/java11vmi.c

```diff
@@ -142,7 +142,10 @@ hashPackageTableDelete(J9VMThread * currentThread, J9ClassLoader * classLoader,
 	rc = hashTableRemove(table, &package);
 
 	if ((U_8 *) package.packageName != (U_8 *)buf) {
-		j9mem_free_memory((void *) package.packageName);
+		if (NULL != package.packageName) {
+			j9mem_free_memory((void *) package.packageName);
+			package.packageName = NULL;
+		}
```

Comment:
While reviewing the implementation of [j9mem_free_memory -> omrmem_free_memory](https://github.com/eclipse-omr/omr/blob/4a95a3754e3f6b74e865b14c32c3cd2556aedb03/port/common/omrmemtag.c#L201-L212), I noticed that there is a NULL check inbuilt in `j9mem_free_memory`. So, we don't need a null check outside `j9mem_free_memory`. We just need to set `packageName` to NULL after invoking `j9mem_free_memory`.
```suggestion
		j9mem_free_memory((void *) package.packageName);
		package.packageName = NULL;
```